app/views/shared/summary_cards/_resource.html.slim

```diff
@@ -23,13 +23,11 @@
       .clearfix
     .summary-card__row
       p
-        - if resource.url?
-          = link_to resource.url, resource.url, target: '_blank'
```

Comment:
Because URLs resources can't have descriptions yet. When you create a new one, you can only give it a name and a URLs. I think we didn't have the View Original button at that time so I was showing the URLs as a description ;) The "View Original" link won't be removed.